ext/crates/algebra/src/algebra/combinatorics.rs

```diff
@@ -11,7 +11,7 @@ pub const MAX_XI_TAU: usize = MAX_MULTINOMIAL_LEN;
 /// If p is the nth prime, then `XI_DEGREES[n][i - 1]` is the degree of $ξ_i$ at the prime p divided by
 /// q, where q = 2p - 2 if p != 2 and 1 if p = 2.
 const XI_DEGREES: [[i32; MAX_XI_TAU]; NUM_PRIMES] = {
-    let mut res = [[0; 10]; 8];
+    let mut res = [[0; MAX_XI_TAU]; NUM_PRIMES];
```

Comment:
This file change looks to me like it could be a separate quick fix commit.

Reply:
That's a good point, I just pushed the change to master

Reply:
Thanks!